src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -923,6 +923,289 @@ public void SarifTransformerTests_ToVersionOne_NotificationExceptionWithStack()
             VerifyCurrentToVersionOneTransformation(V2LogText, V1LogExpectedText);
         }
 
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_ResultLocations()
+        {
+            const string V2LogText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner""
+      },
+      ""logicalLocations"": {
+        ""html.body.header"": {
```

Comment:
I don't think this is a good example for logical location. A `body` might have multiple `header` elements. To locate the one you want, you'd need an XPath expression like `/html/body/header[2]`. I suggest that you make up an example using regular code, for instance, pick your favorite [C# compiler error](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages).

Reply:
I'll just use the collections::list::add set used elsewhere